compatibility/jersey/src/main/java/org/projectnessie/tools/compatibility/jersey/JerseyServer.java

```diff
@@ -172,20 +174,24 @@ private static void withClass(
 
   private static void withJavaxClass(
```

Comment:
Since we are moving towards jakarta and eventually javax will go away, it would be easier to understand if we inverted the logic:

```java
withJakartaClass(
    "org.projectnessie.services.restjakarta.ContentKeyParamConverterProvider",
    config::register,
    true);

private static void withJakartaClass(
    String jakartaClassName, Consumer<Class<?>> whenClassExists, boolean mandatory) {
  Iterator<String> classNames =
      List.of(jakartaClassName, jakartaClassName.replace("restjakarta.", "restjavax."))
          .iterator();
// ...
```

Reply:
Are we sure we need this replacement: `javaxClassName.replace("restjavax.", "rest.")` ? I removed it, and the build passed.

Reply:
Some older Nessie versions required it (a while ago). Maybe no longer needed.